### ISSUE TITLE
Mouse pointer location offset when using overscan on HDMI TVs (workaround)

### ISSUE THREAD

**Issue by sakaki-** [sakaki-/gentoo-on-rpi3-64bit#17]
Hi,
the current 1.1.0 version of the image has a display issue when connected to some HDMI TVs, because (compared to earlier releases) the `disable_overscan=1` line has been commented out (in the `config.txt` file in the top-level directory of the image's first partition's filesystem; also accessible as `/boot/config.txt` when logged in). This was done to move the `config.txt` closer to the standard RPi3 defaults; however, it has an unfortunate side-effect when displaying on an HDMI TV (computer monitor users are unaffected).

Specifically, with TVs the PI will (by default, and with the `disable_overscan=1` line commented), display the active desktop in a smaller area surrounded by a black overscan 'bezel', to ensure that none of the desktop is cropped by the edges of the screen. Unfortunately, the VC4 graphics driver does not appear to take this into account when performing its hardware cursor blitting, with the result that the mouse cursor is offset on the screen from its "true" position, which is very annoying.

To work around this, you can do one of two things:
1. _Either_, uncomment the line `disable_overscan=1` in the file `config.txt` and reboot. This is the simplest option; however, depending on your TV you may experience some slight clipping.
2. _Or_, leave the `disable_overscan=1` line commented out, but switch to using software cursor drawing instead. To do this, create the file `/etc/X11/xorg.conf.d/60-swcursor.conf`, and put in it:
```
Section "Device"
    Identifier "graphicsdriver"
    Driver "modesetting"
    Option "SWcursor" "true"
EndSection
```

Then issue `/etc/init.d/xdm restart` to have the change taken up. If you do move to an SW cursor, you may also find it beneficial to turn off display compositing. To do this, Select <kbd>Applications</kbd>&rarr;<kbd>Settings</kbd>&rarr;<kbd>Window Manager Tweaks</kbd>, select the `Compositor` tab, and untick `Enable display compositing`.

To repeat: **computer monitor users should be unaffected by this issue** (as will those with TVs that report themselves to be monitors, as some will). I'll revert to the uncommented `disable_overscan=1` entry when I do a maintenance release.

Apologies for any inconvenience caused, Sakaki

**Comment by sakaki-** [comment]
This should now be fixed in [v1.1.1](https://github.com/sakaki-/gentoo-on-rpi3-64bit/releases/tag/v1.1.1) (just released), but I'll leave this issue open for a while, in case anyone still on the v1.1.0 encounters the same problem.

**Comment by sakaki-** [comment]
Closing - anyone who this affects should be on >= v1.1.1 by now.
